### PR TITLE
Issue 96 - privileged mode containers

### PR DESCRIPTION
**Pull request by dabooz**
Signed-off-by: Dave Booz <booz@us.ibm.com>
I had a make a few changes not related to privileged mode in order to ensure that the pages I changed were complete.

### CODE REVIEW

File: docs/user_management/security_privacy.md [hunk 1]
```diff
@@ -60,6 +60,10 @@ A compromised agbot can attempt to propose malicious workload deployments, but t
...
...cases where it is required. In {{site.data.keyword.ieam}} you have the ability to specify that an application service should be deployed with privileged process execution enabled. By default, it is disabled. You must explicitly enable it in the [deployment configuration](https://github.com/open-horizon/anax/blob/master/docs/deployment_string.md) of the respective Service Definition file for each service that needs to run in  this mode. And further, any node on which you want to deploy that service must also explicitly allow privileged mode containers. This ensures that node owners have some control over which services are executing on their edge nodes. For an example of how to enable privileged mode policy on an edge node, see [privileged node policy](https://github.com/open-horizon/anax/blob/master/cli/samples/privileged_node_policy.json). If the service definition or one of its dependencies requires privileged mode, the node policy must also allow privileged mode, or else none of the services will not be deployed to the node. For an indepth discussion of privileged mode see [What is privileged mode and do I need it?](https://wiki.lfedge.org/pages/viewpage.action?pageId=44171856).
```
Comment (joewxboy) [inline docs/user_management/security_privacy.md:65]: The "deployment configuration" URL is https://open-horizon.github.io/anax/deployment_string.html

**Review by joewxboy — COMMENTED** [review]
Changed one URL from pointing to MD file in GitHub repo to corresponding page on documentation web site.

File: docs/user_management/security_privacy.md (hunk 1 above)
Comment (dabooz) [reply docs/user_management/security_privacy.md]: done

**Review by Rene-Ch1 — CHANGES_REQUESTED** [review]

File: docs/api/agbot_api.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ lastupdated: "2021-02-20"
 # Agbot API
 {: #agbot_api}
 
-In {{site.data.keyword.edge_notm}} ({{site.data.keyword.ieam}}), the {{site.data.keyword.horizon}} agbot software runs automatically. Each agbot is responsible for communicating with all of the agents that are registered to run the assigned deployment pattern for the agbot, including any services for the pattern. The agbots negotiate agreements with the agents. With the agbot REST APIs, you can configure the agbot by using `http://localhost:8046`. The `hzn agbot` commands interact with these REST APIs.
+In {{site.data.keyword.edge_notm}} ({{site.data.keyword.ieam}}), the {{site.data.keyword.horizon}} agbot software runs automatically. Each agbot is responsible for communicating with all of the agents that are registered to run services by negotiating agreements with the agents. The `hzn agbot` commands interact with these REST APIs. These APIs are not remotely accessible, they can only be used by processes running on the same host as the agbot.
```
[the remark this answers is not in the document]
Comment (dabooz) [reply docs/api/agbot_api.md:20]: done

**Review by Rene-Ch1 — APPROVED** [review]

**Pull request merged**